policyengine_us/variables/gov/states/tax/income/state_income_tax.py

```diff
@@ -15,6 +15,7 @@ class state_income_tax(Variable):
         "ma_income_tax",
         "md_income_tax",
         # "mo_income_tax",  --- activating will cause circular logic errors
+        # "nd_income_tax",  --- activating will cause circular logic errors
```

Comment:
Where's the circular reference?

Reply:
Described in the first comment in [this pull request](#2042) discussion.